### PR TITLE
UI improvement on Home Page

### PR DESCRIPTION
**Pull request by shivambalwani**
issue #3 :
  Added a typing effect on  "Exploring possibilities" .
@pranshuchittora is it good or i need to add some changes to it?

### CODE REVIEW

File: src/pages/home/index.jsx [hunk 1]
```diff
@@ -11,15 +13,13 @@ function HomePage() {
...
           >
-            Exploring possibilities
+           <TypeWriter typing={0.5}>Exploring possibilities</TypeWriter> 
           </Typography>
           <Typography
             variant="h6"
-            style={{ fontWeight: "200", fontStyle: "italic" }}
+            className={styles.headerbottom__home}
           >
             with React Native Elements
```
Comment (nlok5923) [inline src/pages/home/index.jsx:24]: I think it would be good to make it a link which redirects to RNE main repo.

**Review by Uyadav207 — CHANGES_REQUESTED** [review]
@shivambalwani, 

**Problems**

- Direct CSS used.
- Package-lock.json conflict

**Solution**

- try making CSS modules
-  Delete package-lock.json and commit changes on your PR branch

**Comment by shivambalwani** [comment]
> @shivambalwani,
> 
> **Problems**
> 
> * Direct CSS used.
> * Package-lock.json conflict
> 
> **Solution**
> 
> * try making CSS modules
> * Delete package-lock.json and commit changes on your PR branch

Thanks ! Working on it

**Comment by Uyadav207** [comment]
@shivambalwani , Great

**Comment by shivambalwani** [comment]
> @shivambalwani,
> 
> **Problems**
> 
> * Direct CSS used.
> * Package-lock.json conflict
> 
> **Solution**
> 
> * try making CSS modules
> * Delete package-lock.json and commit changes on your PR branch

I have added CSS modules but package-lock.json conflict is not getting resolved . Do you have any idea how to fix it up?

File: src/pages/home/index.jsx (hunk 1 above)
Comment (tewarig) [reply src/pages/home/index.jsx:24]: yes, it would be a great idea @nlok5923

**Review by tewarig — COMMENTED** [review]
well, we can also directly add the typewriting effect. instead of using another package. using {useEfect, useState} hooks. 
check out this code snippet. 
https://gist.githubusercontent.com/abecus/7d20f05a140e99088d9f37517bd594c3/raw/28e0a625691ee655d097fb68cff09f1dd9168ae1/typewriter.js

**Comment by shivambalwani** [comment]
> well, we can also directly add the typewriting effect. instead of using another package. using {useEfect, useState} hooks.
> check out this code snippet.
> https://gist.githubusercontent.com/abecus/7d20f05a140e99088d9f37517bd594c3/raw/28e0a625691ee655d097fb68cff09f1dd9168ae1/typewriter.js

yes, we can but using this package provides more effects which can be applied quite easily when the user needs.

**Comment by pranshuchittora** [comment]
Typing effect is not required & styles are breaking on the home page (spacing issue)

**Comment by shivambalwani** [comment]
> Typing effect is not required & styles are breaking on the home page (spacing issue)

Ok, I will remove the typing effect .But i am not able to understand where the styles are breaking 🤔

**Comment by pranshuchittora** [comment]
https://deploy-preview-40--rne-playground.netlify.app/#/
Check the section below Explore Now button, compare the same with this -> https://deploy-preview-40--rne-playground.netlify.app/#/

**Comment by shivambalwani** [comment]
@pranshuchittora fixed the style breaking issue and removed the typing effect.plz check.

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "react-native-web-linear-gradient": "^1.1.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.1",
+    "react-typewriter": "^0.4.1",
```
Comment (tewarig) [inline package.json:25]: you did remove the typing effect but forgot to uninstall this package.  
``` npm uninstall  react-typewriter  ```

File: src/pages/home/home.module.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.line{
+    /* background-color: rgb(139, 108, 108); */
+    margin-top: 20px;
+    margin-bottom: 15px;
+    border: none;
+    border-top: 1px solid rgb(226, 226, 226);
+}
+.image{
```
Comment (pranshuchittora) [inline src/pages/home/home.module.css:46]: This is too generic name.

File: src/pages/home/home.module.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    transition: transform 0.6s;
+
+}
+
+.inside_paper{
+    bottom: 0;
+     margin-top: 1rem;
+}
+
+.line{
```
Comment (pranshuchittora) [inline src/pages/home/home.module.css:39]: This is too generic name.

File: src/pages/home/home.module.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+.divider{
+    margin: 3vh 0;
+}
+.grid{
+    justify-content: space-evenly;
+    
+}
+
+    
+.paper{
```
Comment (pranshuchittora) [inline src/pages/home/home.module.css:23]: This is too generic name.

File: src/pages/home/home.module.css [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+.headerbottom__home{
+    font-weight: 200 !important;
+     font-style: italic;
+}
+.explore_now{
+    width: 100%;
+     text-align: center;
+      margin-bottom: 2rem;
+}
+.divider{
```
Comment (pranshuchittora) [inline src/pages/home/home.module.css:14]: This is a too generic name.

**Review by pranshuchittora — CHANGES_REQUESTED** [review]
Left some comments.

**Comment by shivambalwani** [comment]
> Left some comments.

@pranshuchittora i have fixed the style naming, but the checks are failing. I don't know why?

**Comment by pranshuchittora** [comment]
Pls pull the new changes and resolve the conflicts.

**Pull request closed without merging**